tbd_tracker_node:
  ros__parameters:
    tracker:
      frame_id: 'map'
      publishers:
          names: ['track_pub', scene_pub]
          track_pub: # Publisher name
            pub_topic: 'tracks'
            msg_type: 'Tracks3D'
            function: 'PublishTracks'
            queue_size: 10
          scene_pub: # Publisher name
            pub_topic: 'scenes'
            msg_type: 'SceneUpdate'
            function: 'PublishScene'
            queue_size: 10

    object_properties: 
      object_classes: ['bicycle',
            'bus',
            'car',
            'motorcycle',
            'pedestrian',
            'trailer',
            'truck']

      bicycle:
          model_type: 'cvcy'
          yaw_proc_var: 1.
          size_proc_var: [1., 1., 1.]
          vel_proc_var: [.1, .1, .1]  
          length: 0.25
          width: 0.25
          height: 0.5
          create_method: 'count'
          delete_method: 'count'
          n_create_min: 3
          n_delete_max: 2 
      
      bus:
          model_type: 'cvcy'
          yaw_proc_var: 1.
          size_proc_var: [1., 1., 1.]
          vel_proc_var: [.1, .1, .1]  
          length: 0.25
          width: 0.25
          height: 0.5
          create_method: 'count'
          delete_method: 'count'
          n_create_min: 1
          n_delete_max: 2

      car:
          model_type: 'cvcy'
          yaw_proc_var: 1.
          size_proc_var: [1., 1., 1.]
          vel_proc_var: [.1, .1, .1]  
          length: 0.25
          width: 0.25
          height: 0.5
          create_method: 'count'
          delete_method: 'count'
          n_create_min: 1
          n_delete_max: 2 

      motorcycle:
          model_type: 'cvcy'
          yaw_proc_var: 1.
          size_proc_var: [1., 1., 1.]
          vel_proc_var: [.1, .1, .1]  
          length: 0.25
          width: 0.25
          height: 0.5
          create_method: 'count'
          delete_method: 'count'
          n_create_min: 3
          n_delete_max: 2 

      pedestrian:
          model_type: 'cvcy'
          yaw_proc_var: 1.
          size_proc_var: [1., 1., 1.]
          vel_proc_var: [.1, .1, .1]  
          length: 0.25
          width: 0.25
          height: 0.5
          create_method: 'count'
          delete_method: 'count'
          n_create_min: 1
          n_delete_max: 2 

      trailer:
          model_type: 'cvcy'
          yaw_proc_var: 1.
          size_proc_var: [1., 1., 1.]
          vel_proc_var: [.1, .1, .1]  
          length: 0.25
          width: 0.25
          height: 0.5
          create_method: 'count'
          delete_method: 'count'
          n_create_min: 3
          n_delete_max: 2 

      truck:
          model_type: 'cvcy'
          yaw_proc_var: 1.
          size_proc_var: [1., 1., 1.]
          vel_proc_var: [.1, .1, .1]  
          length: 0.25
          width: 0.25
          height: 0.5
          create_method: 'count'
          delete_method: 'count'
          n_create_min: 3
          n_delete_max: 2 